Makefile

```diff
@@ -7,7 +7,7 @@ ARCH ?= $(shell go env GOARCH)
 
 APP_NAME = embedded-cluster
 ADMIN_CONSOLE_CHART_REPO_OVERRIDE =
-ADMIN_CONSOLE_IMAGE_OVERRIDE =
+ADMIN_CONSOLE_IMAGE_OVERRIDE = ttl.sh/automated-11614003868/kotsadm:24h
```

Comment:
This probably shouldn't be commited

Reply:
I need to test with the updated kots image before I merge the kots PR, though
and I would prefer to make sure it works in CI as well as in local dev

Reply:
it won't get _merged_, certainly!